ciao-controller/controller_test.go

```diff
@@ -695,7 +695,12 @@ func TestDetachVolumeFailure(t *testing.T) {
 }
 
 func TestDetachVolumeByAttachment(t *testing.T) {
-	err := context.DetachVolume("invalidTenant", "invalidVolume", "attachmentID")
+	tenant, err := addTestTenant()
```

Comment:
Seeing "invalidTenant" change to "addTestTenant()" I thought there needed to coverage of confirmTenant().  But you're not actually adding confirmTenant() but rather just some additional appropriate usage of that function which should already be validated for good and bad tenants by other distinct test cases.